src/components/Table.tsx

```diff
@@ -50,7 +51,10 @@ const Table: React.FC<Props> = ({tableData, setTableData, gridTemplateColumns, c
               return (
                 <React.Fragment key={idx}>
                   {columnDefinitions.map((definition, i)=>
-                    <span className={rowIsSelected? styles.Selected: styles.NotSelected} key={idx + i}>
+                    <span
+                      className={`${rowIsSelected ? styles.Selected : ''} ${tableData.length > 1 && idx === (tableData.length -1) && i === (columnDefinitions.length - 1) ? actionListStyles.LastItemInTable : ''}`}
```

Comment:
give classname for the last span of the table to change the dropdown element to dropup. For table with only 1 row then it would still stay as a dropdown.